helm/csi-powerstore/values.yaml

```diff
@@ -32,7 +32,7 @@ externalAccess:
 #  IfNotPresent: Only pull the image if it does not already exist on the node.
 #  Never: Never pull the image.
 # Default value: None
-imagePullPolicy: IfNotPresent
+imagePullPolicy: Always
```

Comment:
we have to retain IfNotPresent only right?

Reply:
done